conda_setup.py

```diff
@@ -6,10 +6,9 @@
 from Cython.Build import cythonize
 
 extensions = [Extension('pyscipopt.scip', ['pyscipopt/scip.pyx'],
-                        extra_compile_args=['-UNDEBUG'],
```

Comment:
I suppose this was copied from the other setup.py and there I put for debugging purposes. I don't think we want to run with all asserts and debug lines activated so I removed it.